index.d.ts

```diff
@@ -175,7 +175,8 @@ declare namespace Eris {
   // Application Commands
   interface ApplicationCommand<T extends ApplicationCommandTypes = ApplicationCommandTypes> {
     application_id: string;
-    defaultPermission?: boolean;
+    defaultMemberPermissions?: string;
+    DMPermission?: boolean;
```

Comment:
```suggestion
    dmPermission?: boolean;
```
keys should be camel cased